app/main.py

```diff
@@ -758,6 +758,18 @@ def image_xmp_info(image_id):
     return abort(400, description=f"XMP info not found for {image_id}")
 
 
+@app.route("/image_blv_link/<image_id>", methods=["GET"])
+def image_blv_link(image_id):
+    url = Config.BIOLUCIDA_ENDPOINT + "/image/blv_link/{0}".format(image_id)
+    result = requests.request("GET", url)
+
+    response = result.json()
```

Comment:
Dose it make a difference if this is in a try/catch or not? Most the other endpoints are in one so thought I would ask.

Flask seems to handle the errors even without one though right?